qa/L0_FW_iterators/test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+# used pip packages
+
+pip_packages="torchvision mxnet-cu##CUDA_VERSION##"
+
+pushd ../..
+
+cd dali/test/python
+
+NUM_GPUS=$(nvidia-smi -L | wc -l)
+
+test_body() {
+    if [[ "${CUDA_VERSION}" == "90" ]]; then
```

Comment:
Can you put some comment why this is done on this particular way?

Reply:
done